api/main_endpoints/routes/User.js

```diff
@@ -525,4 +526,39 @@ router.post('/usersValidVerifiedAndSubscribed', function(req, res) {
     });
 });
 
+// Generate an API key for the Messages API if the user does not have an API key; otherwise, return the existing API key
+router.post(('/retrieveAPIKey'), async (req, res) => {
+  if (!checkIfTokenSent(req)) {
+    return res.sendStatus(FORBIDDEN);
+  } else if (!checkIfTokenValid(req)) {
+    return res.sendStatus(UNAUTHORIZED);
+  } else {
```

Comment:
we dont need an else block if the first 2 `if`s are both returning

Reply:
word